github-workflow/aks-deploy.yaml

```diff
@@ -84,7 +84,7 @@ jobs:
       with:
         namespace: 'cluster-baseline-settings'
         manifests: |
-          cluster-manifests/cluster-baseline-settings/ns-cluster-baseline-settings.yaml
+          cluster-manifests/base/cluster-baseline-settings/ns-cluster-baseline-settings.yaml
           cluster-manifests/cluster-baseline-settings/flux.yaml
```

Comment:
Does this (cluster-manifests/cluster-baseline-settings/flux.yaml) still exist?

Reply:
Nope! (sorry for the confusion) I'm already taking care of this as part of the next PR.

Reply:
Done here to honor the feedback ❤️  and align changes as they should be in first place (will take over the conflicts in next PR) | Addressed from [84a9a](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/84a9a58f674a7ef0d5e6a45fe124ca8c93990c59) and [d749b](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/d749b9b42f00a8b7650d00d3531deba57530d1f9)